### PR TITLE
Make tutorial work for python3 users

### PR DESCRIPTION
**Pull request by rvandegrift**
I ran into some issues following the tutorial, here are the fixes I needed to complete it.

Thanks!

### CODE REVIEW

**Comment by leplatrem** [comment]
Thanks for taking the time to do this!

Since the tutorial is meant for new projects, I think we can fully switch to Python 3 and get rid of mentions of Python 2. What do you think?

File: docs/source/tutorial.rst [hunk 1]
```diff
@@ -41,18 +41,31 @@ We'll provide a single CLI client in Python, using Curses.
 Setting up the development environment
 --------------------------------------
 
-Make sure you have **virtualenv** (see http://pypi.python.org/pypi/virtualenv).
+If you are using python 2, make sure you have **virtualenv** (see
```
Comment (leplatrem) [inline docs/source/tutorial.rst]: I would write Python (capital)

File: docs/source/tutorial.rst [hunk 2]
```diff
@@ -192,17 +204,19 @@ then use.
...
             user = {'name': name, 'token': _create_token()}
             request.validated['user'] = user
 
 
-When the validator finds errors, it adds them to the **request.errors**
-mapping, and that will return a 400 with the errors.
+The validators work by filling the **request.validated**
+dictionary. When the validator finds errors, it adds them to the
+**request.errors** dictionary, and that will return a 400 with the
+errors.
```
Comment (leplatrem) [inline docs/source/tutorial.rst]: 👍

**Comment by rvandegrift** [comment]
Sounds good - I've dropped the bit about v2.  That also removes the capitalization issue.

**Comment by leplatrem** [comment]
Thanks @rvandegrift!

**Pull request merged**